shared/js/background/broken-site-report.js

```diff
@@ -88,6 +88,60 @@ const requestCategoryMapping = {
     'ignore-user': 'ignoredByUserRequests'
 }
 
+async function digestMessage (message) {
+    const msgUint8 = new TextEncoder().encode(message)
+    const hashBuffer = await crypto.subtle.digest('SHA-256', msgUint8)
+    const hashArray = Array.from(new Uint8Array(hashBuffer))
+    const hashHex = hashArray
+        .map((b) => b.toString(16).padStart(2, '0'))
+        .join('')
+    return hashHex
+}
+
+async function computeLastSentDay (urlString) {
+    const url = new URL(urlString)
+    const time = new Date()
+    // Round time to nearest day
+    time.setHours(0, 0, 0, 0)
+    // Output time as a string in the format YYYY-MM-DD
+    const dayOutput = time.toISOString().split('T')[0]
```

Comment:
This is incorrect for timezones east of GMT. For example for me (in CEST), the date is set to 11pm yesterday. Can we just skip the `setHours` call and do `new Date().toISOString().split('T')[0]` as a one-liner?

Reply:
Love this so much haha (weird bugs are the best). Cheers will fix.